### PR TITLE
improvement: add specific tags for service utils release

### PR DESCRIPTION
**Pull request by dsinghvi**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -166,7 +162,7 @@ jobs:
           name: Compile
           command: yarn compile
       - run:
-          name: Publish all packages
+          name: Publish all packages # TODO can we exclude service-utils?
```
Comment (dsinghvi) [inline .circleci/config.yml:165]: @zachkirsch is there an easy way to do this? https://github.com/yarnpkg/yarn/issues/7051

**Comment by dsinghvi** [comment]
tagging `0.0.184-service-utils` will just release `service-utils`
tagging `0.0.184-cli` will just release the cli and publish packages 
tagging `0.0.184-fern-typescript` will just release the `fern-typescript` generator

**Comment by dsinghvi** [comment]
https://www.notion.so/usebirch/Assembly-Line-Decomp-ae1c6297feee4e89b83e25a1997c22f8 eventually we can have a UI to manage this

**Comment by zachkirsch** [comment]
@dsinghvi this is OBE with the new fern-ts repo right?

**Pull request closed without merging**